src/components/Description/DescriptionLLM/__snapshots__/test.js.snap

```diff
@@ -0,0 +1,60 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<DescriptionLLM /> should render 1`] = `
+<React.Fragment>
+  <div
+    className="row"
+  >
+    <div
+      className="columns large-12"
+    >
+      <div
+        className="callout warning"
+        style={
+          {
+            "alignItems": "center",
+            "display": "flex",
+          }
+        }
+      >
+        <span
+          className="small icon icon-common icon-exclamation-triangle"
+          style={
+            {
+              "color": "#664d03",
+              "fontSize": "2em",
+              "paddingRight": "1rem",
+            }
+          }
+        />
+         
+        <p>
+          <strong>
+            Unreviewed Protein Family Description
+          </strong>
+          <br />
+          This description has been automatically generated using
+           
+          <Memo(Connect(Link))
+            className="ext"
+            href="https://openai.com/research/gpt-4"
+            target="_blank"
+          >
+            GPT-4
+          </Memo(Connect(Link))>
+          , an AI language model, and has not undergone a thorough review by curators.
+          <br />
+          Please exercise discretion when interpreting the information provided and consider it as preliminary.
+        </p>
+      </div>
+    </div>
+  </div>
+  <Description
+    textBlocks={
```

Comment:
What do you do when the ChatGPT says `The function of the family is not clear.` , but we have an InterPro description? Or did you generate the descriptions only for the unintegrated PANTHER?

Reply:
Written in the [PR message](https://github.com/ProteinsWebTeam/interpro7-client/pull/521#issue-1926561583):

> The logic works for any InterPro entry, or member database signature, but right now we only have descriptions for about 5,500 unintegrated PANTHER families.

So yes, we _only_ have descriptions for unintegrated PANTHER families at the moment. Eventually some of these families will be integrated. The current logic to select which description to display is as follows:

1. If there is a description written by curators, pick this one
   * in the case of InterPro entries, it's the common annotation blocks (created with Pronto)
   * in the case of member database signatures, it's descriptions provided by the database (e.g. Pfam descriptions) that we import when doing member database updates
2. Otherwise, if the entry is an integrated member database signature, use the description of the InterPro entry
3. Otherwise, if the entry has an AI-generated description, use this one

Reply:
Thanks for clarifying the logic, it sounds sensible to me.